cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
On Thu, Jul 21, 2016 at 01:03:31PM -0700, Grant Seltzer Richman wrote:
> > +func onlyExportFlagSpecified(context *cli.Context) bool {
> > +	onlyExportFlags := []bool{
> > +		context.Bool("seccomp-only"),
> > +	}
> > +	for _, flag := range onlyExportFlags {
> > +		if flag {
> > +			return true
>
> It's a more general approach. If we wanted to add a feature to
> export just the configuration for one thing then it can be added
> into this function.

How does the caller specify the thing it's interested in?  And it
looks like you're ignoring --output here [1].  I'd rather extend
SaveToFile and Save to take context.  SaveToFile would pass it through
to Save, and Save would do any --*-only switching internally (without
needing a separate helper function).

[1]: https://github.com/opencontainers/ocitools/pull/128/files#diff-3e0e09fa01dec94676136d9c0cf52025R379